docs/server/v5/server/installation/configuration.md

```diff
@@ -67,9 +67,13 @@ If more than one method is used to configure the server, it might be hard to fin
 
 When you run EventStoreDB with this option, it will print out the effective configuration applied from all available sources (default and custom configuration file, environment variables and command line parameters) and print it out to the console.
 
+::: warning
+If you never run EventStoreDB on Linux as a service, when starting the service executable with `--what-if` option, the server _will create the data and log directories_ owned by the current user. It will prevent the service from running properly.
+:::
+
 ::: detail Click here to see a WhatIf example
 ```
-$ eventstore --what-if
+$ eventstored --what-if
```

Comment:
Should we had 2 versions: linux & windows ?

Reply:
I don't think so. It's a huge output, which is exactly the same.